README.md

```diff
@@ -919,92 +918,75 @@ Use the `.is-*` prefix for state classes that are shared between CSS/SCSS and Ja
 <a name="scss-linting"></a>
 ### Linting
 
-A custom `.sass-lint.yml` file has been setup and included with this documentation to aid in the implementation of these standards. This Linting process runs as a Gulp task - courtesy of the [sass-lint](https://github.com/sasstools/sass-lint) package - when your SCSS is processed. It will provide Warnings and Errors within the console for you to review and correct.
-
-Most items will be flagged as a Warning, but a few items that will generate an Error include:
-
-* Extends before mixins
-* Extends before declarations
-* Mixins before declarations
-* One declaration per line
-* Empty line between blocks
-* Single line per selector
-* No invalid hexs
-* No trailing whitespace
-* No URL domains
-* No URL protocols
-* Declarations before nesting
-* Quote attributes
-* Border zero
-* Hex length
-* Hex notation
-* Leading zero
-* Quotes
-* Zero units
-* Space before colon
-* Space after colon
-* Space before brace
-* Space before bang (!)
-* Space after bang (!)
-* Space between parens
-* Space around operator
-* Trailing semicolon
-* Final newline
+#### Sass-lint (deprecated)
+The sass-lint linter is no longer maintained, so O3 is dropping active support for sass-lint. Previous sass-lint docs [can be viewed here](https://github.com/o3world/o3w-frontend-guidelines/tree/4911816198fe5a3aac95cd31199f00927b80e389#linting)
+
+#### Stylelint
+
+A custom `.stylelintrc.json` file has been setup and included with this documentation to aid in the implementation of these standards. It will provide Warnings and Errors within the console for you to review and correct. This stylelint config extends the stylelint recommended config, and uses an additional [stylelint-order](https://github.com/hudochenkov/stylelint-order) plugin for property sorting.
+
+To see a list of the rules, refer to the [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js) as well as the rules in the `.stylelintrc.json` file in this repository
+
+[Stylelint User Guide](https://stylelint.io/user-guide)
+
+##### Setup
+1. `npm install stylelint stylelint-config-standard stylelint-order --save-dev`
```

Comment:
@sgengler I'm not as familiar with how this would be setup with our build process. is this npm install needed unique or can it be included in a `package.json`?

Reply:
@brianO3world Hey just saw this. `npm install stylelint...etc` will add these packages to `node_modules` and also list them as devDependencies in `package.json`